src/OrchardCore.Modules/OrchardCore.Alias/Module.txt

```diff
@@ -5,6 +5,5 @@ Website: http://orchardproject.net
 Version: 2.0.x
 OrchardVersion: 2.0.x
 Description: The alias module enables content items to have custom logical identifier.
-FeatureDescription: Alias content part.
```

Comment:
This is supposed to be the description of the implicit feature that is created for a module, when there is no `Features:` section. To distinguish from the description of the module itself.

Reply:
I would assume this is what we use right now when we display the features, but as we also have Category and Dependencies and not FeatureCategory and FeatureDependencies I assume it's fine to remove it.